app/models/spree_amazon/order.rb

```diff
@@ -25,6 +27,24 @@ def confirm
     mws.confirm_order
   end
 
+  def close_order_reference!
+    response = mws.close_order_reference
+    parsed_response = Hash.from_xml(response.body) rescue nil
+
+    if response.success
+      true
+    else
+      parsed_response = Hash.from_xml(response.body) rescue nil
+      message = if parsed_response && parsed_response['ErrorResponse']
+        error = parsed_response.fetch('ErrorResponse').fetch('Error')
+        "#{response.code} #{error.fetch('Code')}: #{error.fetch('Message')}"
```

Comment:
Using `fetch` without providing a default value is going to crash when the key is missing, should we be providing an empty array as the default value?